README.md

```diff
@@ -29,6 +29,21 @@ To run in udev mode:
 DEVNAME=/dev/nvme0n1 azure-nvme-id --udev
 ```
 
+### Enabling LUN calculation by Namespace Identifier
+
+Pass AZURE_LUN_CALCULATION_BY_NSID_ENABLED=1 to cmake:
+
+```
+cmake -DAZURE_LUN_CALCULATION_BY_NSID_ENABLED=1 .
```

Comment:
Looking at the code I think this is the default behavior? We should mention that being the case

Reply:
good point, done!